common-util/src/main/java/org/sunbird/common/models/util/ActorOperations.java

```diff
@@ -8,6 +8,7 @@
  */
 public enum ActorOperations {
   ENROLL_COURSE("enrollCourse"),
+  UNENROLL_COURSE("unenrollCourse"),
```

Comment:
UNENROLL_COURSE is there in release-1.11 branch. Are you not having latest release 1.11 changes in your branch?

Reply:
Yes, I am going to re-merge my changes and put another pull-request without such discrepancies.